src/webview/mod.rs

```diff
@@ -160,6 +160,31 @@ pub struct WebViewAttributes {
   /// allow to navigate and false is not.
   pub navigation_handler: Option<Box<dyn Fn(String) -> bool>>,
 
+  /// Set a download started handler to manage incoming downloads.
+  ///
+  /// The closure takes two parameters - the first is a `String` representing the url being downloaded from and and the
+  /// second is a mutable `PathBuf` reference that (possibly) represents where the file will be downloaded to. The latter
+  /// parameter can be used to set the download location by assigning a new path to it - the assigned path _must_ be
+  /// absolute. The closure returns a `bool` to allow or deny the download.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Windows:** The assigned path cannot include a UNC (Universal Naming Convention) prefix. This prefix is
+  /// sometimes included when canonicalizing paths in order to differentiate between networked filesystems.
```

Comment:
We can just use `dunce` crate to strip UNC from the path after it has been modified by the handler